electron-app/src/components/settings/PremiumSettings.vue

```diff
@@ -2,16 +2,25 @@
   <v-row class="premium-settings">
     <v-col>
       <v-card>
-        <v-card-title>Premium Settings</v-card-title>
+        <v-card-title>Rotki Premium</v-card-title>
         <v-card-text>
+          <p>
+            Rotki Premium is an optional subscription service to gain access to
```

Comment:
to gain access to ...  
->
analytics, graphs and unlock many additional features. For more information on what is available visit the Rotki premium website.